include/alpaka/api/unifiedCudaHip/math.hpp

```diff
@@ -50,6 +50,131 @@ namespace alpaka::math::internal
             ALPAKA_UNREACHABLE(T_Arg{});
         }
     };
+
+    //! The CUDA sqrt trait specialization for real types.
+    template<typename T_Arg>
+    requires(std::is_floating_point_v<T_Arg>)
```

Comment:
```suggestion
    template<std::floating_point T_Arg>
```
Please use the concept and apply it to the other functions. You need also to `#include <concepts>`.

Reply:
Done, thanks.